crates/rattler_installs_packages/src/artifacts/sdist.rs

```diff
@@ -264,7 +264,7 @@ mod tests {
             &env_markers,
             None,
             &resolve_options,
-            package_db.1.path(),
+            WheelCache::new(package_db.1.path().join("wheels")),
```

Comment:
Should this be wheels or should it mention something like "local_wheels"? I think we might have to do some pruning on these cached wheels more often vs. ones that come from      `pypi` (e.g. if the system changes).

Reply:
Yeah so w.r.t to artifacts we are actually storing the http request/response pairs that serve as both a check for staleness and such, but also to get the `artifact` be at an `sdist` or `wheel` from. The wheel cache for locally built wheels is seperate, so these will not intersect on disk.